packages/editor-ui/src/api/workflows.ts

```diff
@@ -1,6 +1,11 @@
-import { IRestApiContext } from '@/Interface';
-import { makeRestApiRequest } from './helpers';
+import { IRestApiContext, IWorkflowTemplate } from '@/Interface';
+import { makeRestApiRequest, get } from './helpers';
+import { TEMPLATES_BASE_URL } from '@/constants';
 
 export async function getNewWorkflow(context: IRestApiContext, name?: string) {
 	return await makeRestApiRequest(context, 'GET', `/workflows/new`, name ? { name } : {});
-}
\ No newline at end of file
+}
+
+export async function getWorkflowTemplate(templateId: string): Promise<IWorkflowTemplate> {
+	return await get(TEMPLATES_BASE_URL, `/workflows/template/${templateId}`);
```

Comment:
should be this. We already have the hint in the constant 🙈

```suggestion
	return await get(TEMPLATES_BASE_URL, `/workflows/templates/${templateId}`);
```

Reply:
endpoint is already in production.. so cannot change this at this point.

Reply:
updated now